ffi/build.ps1

```diff
@@ -0,0 +1,23 @@
+# Set the name of your Go program
```

Comment:
Library, not program

Reply:
fixed - although a library is a program ;-)